packages/gatsby/src/utils/show-experiment-notice.ts

```diff
@@ -26,7 +47,7 @@ export function showExperimentNoticeAfterTimeout(
   }
 
   const noticeTimeout = setTimeout(() => {
-    reporter.info(`\n\n${noticeText}\n\n`)
+    noticesToShow.push({ noticeText, umbrellaLink, experimentIdentifier })
 
     getConfigStore().set(configStoreKey, Date.now())
```

Comment:
Should we move setting the key to when we actually print message? Setting it here doesn't guarantee that message will be shown in case there is fatal error or user interrupt with ctrl+c or something like that

Reply:
👍